src/components/SQForm/SQFormDatePicker.tsx

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import moment from 'moment';
```

Comment:
If we're only using the `Moment` type you could simplify things a bit and just destructure that here rather than using `moment.Moment`
```suggestion
import {Moment} from 'moment';
```